onnx/test/cpp/function_context_test.cc

```diff
@@ -78,6 +79,9 @@ BuildFloatFunctionBody(const FunctionBodyBuildContext& ctx, const OpSchema& sche
 
 void RegisterCustomFuncFloatSchema() {
   ONNX_NAMESPACE::OpSchema schema;
+#ifdef __ONNX_DISABLE_STATIC_REGISTRATION
```

Comment:
why do you need this?

Reply:
I was also considering whether to revert it... Just in case if someone wants to test C++ ONNX with ONNX_DISABLE_STATIC_REGISTRATION and bumps into issue here. However, even if that is the case, the whole test will still fail with schema_test.cc anyway... So I will remove it. Thanks.

Reply:
what would be the issue in that case?

Reply:
oh you mean this test and schema_registration_test cannot go together?

Reply:
Yes, then it will still fail so the if condition here is useless